### ISSUE TITLE
2600/7800 cores fail to launch (Atari7800_20221227.rbf)

### ISSUE THREAD

**Issue by davetansley** [MiSTer-devel/Atari7800_MiSTer#23]
I've noticed that the Atari 2600/7800 cores fail to run on my MiSTer currently.

When I launch the cores, my display (Sony Bravia connected via HDMI) tells me that no signal is being received. The display is normally pretty tolerant of non-standard frequencies. I can still SSH into the MiSTer and it seems to still be running, it's just not outputting a display.

If I revert to the previous version (Atari7800_20220212.rbf) it launches as expected.

I've tried deleting any previous config for the core. I have no Atari7800 specific config in MiSTer.ini.

Is this a known issue?

**Comment by birdybro** [comment]
What is your vsync_adjust= set to in your MiSTer.ini currently?

If you have vsync_adjust=2 or 1 on and you don't have "Stabilize Video" enabled, this can happen sometimes with most tv's, especially on vsync2.

**Comment by davetansley** [comment]
Currently:

vsync_adjust=1

I also tried removing the MiSTer.ini file so that it would just use the defaults (not sure what vsync_adjust defaults to). Had the same issue.

The same setting was used in the earlier version of the core, and it worked then.

**Comment by davetansley** [comment]
Confirmed that the issue also happens with vsync_adjust explicitly set to 0.

**Comment by trebor68** [comment]
SainT mentioned, in the FPGA implementation of MARIA under his 7800GD cart, which is also a branch from the MiSTer tree, that the Sony Bravia specifically is "unbelievably picky" in one [post](https://forums.atariage.com/topic/256916-the-7800gd-new-7800-flash-cartridge/?do=findComment&comment=5194383) and "EXTREMELY picky" in [another](https://forums.atariage.com/topic/256916-the-7800gd-new-7800-flash-cartridge/?do=findComment&comment=5280883).

Posting some of the notes here too for convenience and posterity:

_"**...many HDMI TV's (like my Bravia's for example) are EXTREMELY picky with sync jitter to the point they just wont display a picture. The 7800 sync is very jittery due to the crazy clock cycle stretching Maria does to handle slow access to different parts of the system.**_

_The RAD2X is a scanline based upscaler, so it can do nothing to help sort out sync issues. My monitor, for example, doesn't care about this and will display a picture just fine (via OSSC in this case, same type of upscaler), but as mentioned **the Bravia just wont work.** I use a Tink5xp in framebuffer mode and have tested with component and RGB inputs from the 7800GD and it works perfectly, even with the Bravia's._

_So, basically, it depends on your setup -- it may work just fine, it may not. If you already have a RAD2X, definitely test it before looking elsewhere, but **you may find you need a framebuffer based upscaler**. "_


Also...
_"This is why I recommend an upscaler which has triple buffer capability (like tink 5xp) for maximum compatibility. If your TV is happy with hdmi jitter (my monitor doesn’t care at all), then you can simply use frame lock, but if it’s not (my bravia is unbelievably picky) then you’ll need to **switch to triple buffer to get a picture**. If you’re using a CRT, then it’s all good. CRTs just work."_
